app/src/main/java/org/mifos/mobilebanking/ui/fragments/RecentTransactionsFragment.java

```diff
@@ -147,11 +147,23 @@ public void onRefresh() {
         recentTransactionsPresenter.loadRecentTransactions(false, 0);
     }
 
+    /**
+     * Shows a SnackBar message using utils.Toaster class
+     * @param message
+     */
     @Override
     public void showMessage(String message) {
         Toaster.show(rootView, message, Toaster.LONG);
     }
 
+    /**
+     * Shows a Toast message
+     * @param message
+     */
+    public void showToastMessage(String message) {
```

Comment:
Don't create new method. 

Use this method provided [here](https://github.com/openMF/mifos-mobile/blob/development/app/src/main/java/org/mifos/mobilebanking/ui/activities/base/BaseActivity.java#L110).